app/assets/stylesheets/hyrax/sidebar.scss

```diff
@@ -36,8 +36,13 @@ $gutter-width: $grid-gutter-width/2;
       transform: rotate(-180deg);
     }
 
-    .nav-pills > li > a {
-      padding-left: 24px;
+    .nav-pills > li {
+      a {
+        padding-left: 24px;
+      }
+      ul > li a {
```

Comment:
Decided to make menu items indent properly, porting and adapting Hyku work https://github.com/projecthydra-labs/hyku/pull/1049, since I needed to add more indented items to the dashboard menu.